projects/fhi-angular-highcharts/src/lib/fhi-angular-highcharts.component.html

```diff
@@ -1,8 +1,8 @@
-<article *ngIf="diagramOptions !== undefined">
-  <h1 class="h3 mb-3">{{ diagramOptions.title }}</h1>
+<article *ngIf="diagramOptionsInternal !== undefined">
```

Comment:
@proand The user might send diagramOption as `null`. Maybe change `diagramOptionsInternal !== undefined` 
`@if(diagramOptionsInternal)` or
`@if(diagramOptionsInternal != null)` or
`@if(diagramOptionsInternal !== null || diagramOptionsInternal !== undefined)`

Reply:
I have a little list (not exhaustive) of values, or missing values, in the diagramOptions that can give errors in current implementation. I'm pondering making a test of the input, and then jsut test for `!diagramOptionsValid` or something like that... but probably best to create an issue for that, and go for your solution now.